securedrop/tests/common.py

```diff
@@ -1,120 +1,272 @@
+# -*- coding: utf-8 -*-
+
+from functools import wraps
+import gnupg
+import mock
 import os
 import shutil
-import uuid
 import subprocess
-
-import gnupg
+import time
```

Comment:
[In PEP8](https://www.python.org/dev/peps/pep-0008/#imports), standard library imports come before third party imports (e.g. `time` before `gnupg`)